### ISSUE TITLE
documentation

### ISSUE THREAD

**Issue by dacresni** [strathausen/dracula#3]
Could you wright a step by step tutorial on how you got your example? I looked at this project because I wanted a replacement for graphviz in my app. I wanted to just add edges to a graph similar to the manor you add edges in your example on your blog. its very much like graph viz

**Comment by strathausen** [comment]
Good point, I guess it's the single most pressing issue right now. On it.
